src/App.css

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
This file should only contain global css. If some styles are not needed globally, we should stick to css modules instead.